design-doc.md

```diff
@@ -0,0 +1,137 @@
+# Virtualship Design Document
+
+> Document created as a result of [this discussion topic](https://github.com/OceanParcels/virtualship/discussions/187).
+
+## Essence of Virtualship
+
+Virtualship interpolates hydrodynamical fields the way instruments would. Users can combine these observations into an expedition ("cruise"). Users can also deploy individual instruments in the flowfield, either as part of an expedition or independently, to compare against actual observations.
```

Comment:
Suggestion to highlight the two types of users
```suggestion
Virtualship interpolates hydrodynamical and biogeographical fields the way instruments would. **Student Users** can combine these observations into an expedition (also known as a "cruise"). **Research Users** can also deploy individual instruments in these field, either as part of an expedition or independently, to compare against actual observations.
```

Reply:
Yes, I forgot to mention: Should we settle a bit here on terminology? "Expedition" or "cruise"?